### PR TITLE
Collect Route

### PR DESCRIPTION
**Pull request by elguaposalsero**
Added the "collect" route, and used the router inside of app.js. Also added nodemon to simplify development.

### CODE REVIEW

File: controllers/collect.js [hunk 1]
```diff
@@ -1,2 +1,21 @@
 // Listening for requests from webhooks
 // Sends the requests to our PG and Mongo databases
+
+// /api/collect/
+
+Request = require("../models/request");
```
Comment (lukeoguro) [inline controllers/collect.js]: I believe line 6 is redundant.

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node index.js"
+    "start": "nodemon index.js"
```
Comment (lukeoguro) [inline package.json]: Could you make this `node run dev` instead? So when we actually do `npm start` on deployment, it doesn't use `nodemon`.

**Review by lukeoguro — CHANGES_REQUESTED** [review]
Minor changes — hmu with anything

File: controllers/collect.js (hunk 1 above)
Comment (elguaposalsero) [reply controllers/collect.js]: Yep you're right, changed it

File: package.json (hunk 2 above)
Comment (elguaposalsero) [reply package.json]: Done

**Review by westrachel — APPROVED** [review]
these updates look good

**Review by lukeoguro — APPROVED** [review]
Looks good!

**Pull request merged**